widget_pages/dashboard.py

```diff
@@ -19,25 +20,104 @@
 import pyqtgraph as pg
 
 from widget_pages.toolbar import ToolBar
+from widget_pages.sidebar import SideBar
+
+
+# class TabShowGraph(QWidget):
```

Comment:
What is going on here? is this just for the fake graph?

Reply:
Note, Angus uses TabShowGraph in https://github.com/liu-allan/Leukemia-Treatment-Application/pull/68/files

Reply:
Because of this, I think we should wait for him to merge his first

Reply:
shouldn't this code be uncommented then? If Angus merges first, this PR will just comment the code out and break things. @xieanne @angusYuhao

Reply:
this is because i added the sidebar and the layout doesn't match if i have that function. but i can try to re-add that function in